spec/features/users_creates_new_account_spec.rb

```diff
@@ -1,5 +1,28 @@
 require "rails_helper"
 
+feature "admin user accesses #index" do
+  scenario "successfully" do
+    user = create(:user, admin: true)
+    feature_log_in_as user
+
+    visit users_path
+
+    expect(page).to have_text(I18n.t("users.index.title"))
+  end
+end
+
+feature "non-admin user accesses #index" do
```

Comment:
This should be moved to a controller spec

Reply:
I don't seem to be able to access methods in the application controller from the controller specs?